src/Uno.UI/ts/Windows/Storage/Pickers/FileOpenPicker.ts

```diff
@@ -8,7 +8,10 @@
 		public static async nativePickFilesAsync(
 			multiple: boolean,
 			showAllEntry: boolean,
-			fileTypesJson: string): Promise<string> {
+			fileTypesJson: string,
+			id: string,
+			startIn: StartInDirectory
```

Comment:
Addresses the warning

```suggestion
			startIn: StartInDirectory,
```

![image](https://user-images.githubusercontent.com/31348972/129492098-7c854c48-9c59-4f04-97c9-04ad1a245667.png)


Also makes smaller diff in the future if a new parameter is added.